### ISSUE TITLE
The website doesn't explain how to import a package

### ISSUE THREAD

**Issue by lucamug** [elm/package.elm-lang.org#334]
I have seen beginners struggling in understanding how to import a package, especially because there is no enforced consistency in naming.

For example
```
elm/html              -> import Html
evancz/elm-playground -> import Playground
mdgriffith/elm-ui     -> import Element
```
Such information now is available in the right side column but it is not explicit:

<img width="924" alt="Screen Shot 2020-04-12 at 8 45 32" src="https://user-images.githubusercontent.com/5551094/79058061-d697af80-7ca3-11ea-843e-af534cdeda37.png">

This extra info could be added in the already present "About" page (https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/about):

<img width="910" alt="Screen Shot 2020-04-12 at 8 50 00" src="https://user-images.githubusercontent.com/5551094/79058072-e0211780-7ca3-11ea-83c7-dd9cc54f38ac.png">

**Comment by Kurren123** [comment]
+1 I struggled with this quite a bit when starting off

**Comment by rlefevre** [comment]
I think the risk is high that beginners will copy the whole thing indeed, even more because it will work.

Also, with regard to the examples above, the 4th line of `elm/html`  documentation shows an import example:
https://package.elm-lang.org/packages/elm/html/latest/

Idem for `elm-ui` in the example just after the introduction:
https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/

If this did not help, why adding them again in the `About` page would?

In my opinion, they are better illustrated by code examples that should include the `import` directives to show why the names are qualified or not.

**Comment by lucamug** [comment]
Yes, sometime package authors add example that also include the imports, but it is not always the case, so I think an example generated automatically could be a safer approach.

Having begineers copying all the imports, even if not necessary, seems to me a minor issue compared to have them struggling in making their code to work. At least the code is running. They will eventually learn and most IDE now have wornings for unused imports.

I agree that adding also a link to an explanation would be good. This could be a good target: https://elm-lang.org/docs/syntax#modules
